dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-body.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<title>non-passive mousewheel event listener on body</title>
+<link rel="help" href="https://github.com/WICG/interventions/issues/64">
```

Comment:
We should ideally have a real spec for this instead of a closed interventions issue...

Reply:
We can now point to https://w3c.github.io/uievents/#cancelability-of-wheel-events and https://github.com/w3c/uievents/issues/331